src/webhooks/webhooks.js

```diff
@@ -26,16 +27,16 @@ class Webhooks {
    * @throws {Error} throw error when a webhook signature verification fails.
    *
    * @example
-   * const Mux = require('mux');
-   * const { Webhooks } = Mux;
-   *
+   * const Mux = require('@mux/mux-node');
+   * const muxClient = new Mux(accessToken, secret);
+   * const { Webhooks } = muxClient;
```

Comment:
Good catch on the `required` fix!
 
But I think `Webhooks` is a top-level util, like `JWT`, likewise it's a property on the `Mux` constructor. (The API access token and secret are not needed to verifyHeaders, the signingSecret is needed)

```js
const Mux = require('@mux/mux-node');
const { Webhooks } = Mux

Mux.Webhooks.verifyHeader(payload, header, signingSecret);
```

Reply:
Updated

Reply:
I think for V2 we may want to move this to `System` to better reflect our actual API.